.github/workflows/wordlist.yml

```
@@ -0,0 +1,33 @@
+name: Sort Wordlist
+
+on:
+  pull_request:
+    paths:
+      - '.wordlist.txt'
+
+jobs:
+  sort-wordlist:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Sort .wordlist.txt
+        run: |
+          LC_ALL=C sort .wordlist.txt -o .wordlist.txt
+          if ! git diff --quiet .wordlist.txt; then
+            git config user.name "github-actions[bot]"
+            git config user.email "github-actions[bot]@users.noreply.github.com"
+            git add .wordlist.txt
+            git commit -m "chore: sort .wordlist.txt"
+          fi
+
+      - name: Push changes
+        if: github.event_name == 'pull_request'
+        run: |
+          git push origin "HEAD:${{ github.head_ref }}"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Probably no need to check for `pull_request` because we have `on: pull_request` trigger, and we can combine both steps.

```suggestion
      - name: Sort .wordlist.txt
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        run: |
          LC_ALL=C sort .wordlist.txt -o .wordlist.txt
          if ! git diff --quiet .wordlist.txt; then
            git config user.name "github-actions[bot]"
            git config user.email "github-actions[bot]@users.noreply.github.com"
            git add .wordlist.txt
            git commit -m "chore: sort .wordlist.txt"
            git push origin "HEAD:${{ github.head_ref }}"
          fi
```